pkg/deploy/spindeploy/transformer/telemety.go

```diff
@@ -0,0 +1,114 @@
+package transformer
+
+import (
+	"context"
+	"github.com/armory/spinnaker-operator/pkg/apis/spinnaker/v1alpha2"
+	"github.com/armory/spinnaker-operator/pkg/generated"
+	"github.com/armory/spinnaker-operator/pkg/inspect"
+	"github.com/armory/spinnaker-operator/version"
+	"github.com/ghodss/yaml"
+	"github.com/go-logr/logr"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	EchoConfigFile     = "echo.yml"
```

Comment:
If these constants are not used outside of this file, you can make them private by starting with lowercase (`echoConfigFile` instead of `EchoConfigFile`).